evalml/tests/component_tests/test_exponential_smoothing_regressor.py

```diff
@@ -0,0 +1,196 @@
+        == ModelFamily.EXPONENTIAL_SMOOTHING
+    )
+
+
+def test_problem_types():
+    assert set(ExponentialSmoothingRegressor.supported_problem_types) == {
+        ProblemTypes.TIME_SERIES_REGRESSION
+    }
+
+
+def test_model_instance(ts_data):
+    X, y = ts_data
+    clf = ExponentialSmoothingRegressor()
+    fitted = clf.fit(X, y)
+    assert isinstance(fitted, ExponentialSmoothingRegressor)
+
+
+def test_fit_ts_without_y(ts_data):
+    X, y = ts_data
+
+    clf = ExponentialSmoothingRegressor()
+    with pytest.raises(
+        ValueError, match="Exponential Smoothing Regressor requires y as input."
+    ):
+        clf.fit(X=X)
+
+
+@pytest.mark.parametrize("train_features_index_dt", [True, False])
+@pytest.mark.parametrize("train_target_index_dt", [True, False])
+@pytest.mark.parametrize(
+    "train_none, no_features, datetime_feature",
+    [
+        (True, False, False),
+        (False, True, False),
+        (False, False, True),
+        (False, False, False),
+    ],
+)
+@patch("sktime.forecasting.exp_smoothing.ExponentialSmoothing.fit")
+def test_remove_datetime(
+    mock_fit,
+    train_features_index_dt,
+    train_target_index_dt,
+    train_none,
+    datetime_feature,
+    no_features,
+    get_ts_X_y,
+):
+    X_train, _, y_train = get_ts_X_y(
+        train_features_index_dt,
+        train_target_index_dt,
+        train_none,
+        datetime_feature,
+        no_features,
+        test_features_index_dt=False,
+    )
+
+    if not train_none:
+        if train_features_index_dt:
+            assert isinstance(X_train.index, pd.DatetimeIndex)
+        else:
+            assert not isinstance(X_train.index, pd.DatetimeIndex)
+        if datetime_feature:
+            assert X_train.select_dtypes(include=["datetime64"]).shape[1] == 1
+    if train_target_index_dt:
+        assert isinstance(y_train.index, pd.DatetimeIndex)
+    else:
+        assert not isinstance(y_train.index, pd.DatetimeIndex)
+
+    clf = ExponentialSmoothingRegressor()
+    clf.fit(X_train, y_train)
+
+    y_train_removed = mock_fit.call_args.kwargs.get("y", None)
+    if y_train_removed is not None:
+        assert not isinstance(y_train_removed.index, pd.DatetimeIndex)
+
+
+def test_set_forecast(get_ts_X_y):
+    from sktime.forecasting.base import ForecastingHorizon
+
+    _, X_test, _ = get_ts_X_y(
+        train_features_index_dt=False,
+        train_target_index_dt=False,
+        train_none=False,
+        datetime_feature=False,
+        no_features=False,
+        test_features_index_dt=False,
+    )
+
+    clf = ExponentialSmoothingRegressor()
+    fh_ = clf._set_forecast(X_test)
+    assert isinstance(fh_, ForecastingHorizon)
+    assert len(fh_) == len(X_test)
+    assert fh_.is_relative
+
+
+def test_feature_importance(ts_data):
+    X, y = ts_data
+    clf = ExponentialSmoothingRegressor()
+    with patch.object(clf, "_component_obj"):
+        clf.fit(X, y)
+        assert clf.feature_importance == np.zeros(1)
```

Comment:
Filing [this](https://github.com/alteryx/evalml/issues/3183) to standardize the way we handle these not-supported feature importance tests.